qemu/tests/cfg/virtio_mem_numa_basic.cfg

```diff
@@ -40,3 +40,7 @@
     variants operation:
         - @default:
         - with_reboot:
+        - with_migration:
+            kill_vm_on_error = yes
+            mig_timeout = 1200
+            migration_protocol = "tcp"
```

Comment:
Could you please check the end of this file ?   looks unknown character is there.

Reply:
Sorry, the new line was missing, changed now!